AirBench/Controllers/HomeController.cs

```diff
@@ -22,9 +20,17 @@ public HomeController()
         [HttpGet]
         public ActionResult Index()
         {
-            var repository = new UserRepository(context);
-            IList<User> users = repository.GetUsers();
-            return View(users);
+            if (Request.IsAuthenticated)
+            {
+                User currentUser = new UserRepository(context).GetLoggedInUser(User.Identity.Name);
+                if(currentUser != null)
+                {
+                    ViewBag.Name = currentUser.UserName;
```

Comment:
It looks like you used the user's `UserName` when setting the auth cookie in the `AccountController`, so you should be able to just use `User.Identity.Name` here instead of retrieving the user record from the database only to use the `UserName` property value.